### PR TITLE
fix(shell): correct /usage remaining quota display

### LINKED ISSUE #2019: Wrong usage color
### What version of Kimi Code CLI is running?

1.37.0

### What issue are you seeing?

It uses wrong color in the usage command.

The color is chosen by the remaining ratio:

https://github.com/MoonshotAI/kimi-cli/blob/34025f6af57c60de5b039673dfd607f18e2e903b/src/kimi_cli/ui/shell/usage.py#L255-L257
https://github.com/MoonshotAI/kimi-cli/blob/34025f6af57c60de5b039673dfd607f18e2e903b/src/kimi_cli/ui/shell/usage.py#L276-L281

### What is the expected behavior?

```diff
+ # remaining ratio is passed in
  def _ratio_color(ratio: float) -> str:
-     if ratio >= 0.9:
+     if ratio <= 0.1:
          return "red"
-     if ratio >= 0.7:
+     if ratio <= 0.3:
        return "yellow"
     return "green"
```

### PR DESCRIPTION
**Pull request by wbxl2000**
## Summary

Fixes #2019.

This updates `/usage` so the displayed `% left` label, progress bar fill, and warning color all use the remaining quota ratio consistently. High remaining quota now renders as green/full, while near-exhausted quota renders yellow or red.

Thanks to #2039, #2046, and #1411 for the related reports and proposed fixes that helped converge on the behavior here.

## Changes

- Render the quota bar using remaining quota instead of used quota.
- Flip `_ratio_color` thresholds to treat low remaining quota as dangerous.
- Clamp unusual API values so over-used or negative-used rows do not overfill the bar.
- Add regression tests for remaining quota colors, progress fill, zero-limit, over-used, and reset hint rendering.
- Add release-note entries in English and Chinese changelogs.

## Tests

- `uv run ruff check src/kimi_cli/ui/shell/usage.py tests/ui/test_usage.py`
- `uv run ruff format --check src/kimi_cli/ui/shell/usage.py tests/ui/test_usage.py`
- `uv run pytest tests/ui/test_usage.py`
- `make check-kimi-cli`

Note: `make check-kimi-cli` exits 0. `ty check` is non-blocking in the Makefile and still reports existing repo-wide diagnostics unrelated to this change.

### CODE REVIEW

**Comment by wbxl2000** [comment]
image

**Comment by wbxl2000** [comment]
@codex

File: src/kimi_cli/ui/shell/usage.py [hunk 1]
```diff
@@ -253,14 +253,19 @@ def _build_usage_panel(summary: UsageRow | None, limits: list[UsageRow]) -> Pane
...
+        remaining = min(max(row.limit - row.used, 0), row.limit)
+        remaining_ratio = remaining / row.limit
+    else:
+        remaining = 0
+        remaining_ratio = 0
+    color = _ratio_color(remaining_ratio)
 
     label = Text(f"{row.label:<{label_width}}  ", style="cyan")
-    bar = ProgressBar(total=row.limit or 1, completed=row.used, width=20, complete_style=color)
+    bar = ProgressBar(total=row.limit or 1, completed=remaining, width=20, complete_style=color)
```
[the remark this answers is not in the document]
Comment (wbxl2000) [reply src/kimi_cli/ui/shell/usage.py]: Fixed by setting `finished_style=color` on the `ProgressBar`, and the test now covers the 100% remaining case via Rich public rendering output.

File: tests/ui/test_usage.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
+from __future__ import annotations
+
+import pytest
+
+from kimi_cli.ui.shell.usage import UsageRow, _format_row, _ratio_color
+
+
+def _extract_cells(table):
+    return [column._cells[0] for column in table.columns]
```
[the remark this answers is not in the document]
Comment (wbxl2000) [reply tests/ui/test_usage.py]: Adjusted the tests to avoid Rich private `column._cells`; they now inspect `Console.render_lines()` output through the public rendering path.

File: src/kimi_cli/ui/shell/usage.py [hunk 3]
```diff
@@ -253,14 +253,19 @@ def _build_usage_panel(summary: UsageRow | None, limits: list[UsageRow]) -> Pane
...
```
[the remark this answers is not in the document]
Comment (wbxl2000) [reply src/kimi_cli/ui/shell/usage.py:267]: Fixed by computing a positive `bar_total` for non-positive limits, so negative API values render as 0% left with an empty bar. Added coverage for negative `limit` as well.

**Comment by wbxl2000** [comment]
@codex
